main.js

```diff
@@ -114,6 +114,20 @@ updateNotifier({pkg: cli.pkg}).notify();
 
 const {input, flags: opts} = cli;
 
+// Make data types for `opts.space` match those of the API
+if (typeof opts.space === 'string') {
```

Comment:
See my other comments regarding this code.
In addition the cases where `yargs-parser` set `space` as `'true'` or `'false'` is not handled.
I think would be clearer and cover the `'true'` or `'false'` case:
```js
if (typeof opts.space !== 'undefined') {
	if (opts.space === 'true') {
		opts.space = true;
	} else if (opts.space === 'false') {
		opts.space = false;
	} else if (/^\d+$/.test(opts.space)) {
		opts.space = parseInt(opts.space, 10);
	}
	if (opts.space) {
		input.push(opts.space);
	}
	opts.space = true;
}
```

Reply:
That's a great idea, handling those `'true'` and `'false'` cases! I'll get on that. I'd do a slightly different implementation, though, since that quick example would always set the value to `true` and would push numbers and `true` to `input`.

Reply:
If we add the `+` to the regexp then the numbers wouldn't be parsed as the `input`.
Indeed the `opts.space = true;` is wrongly positioned. It should probably be something like that:
```js
if (typeof opts.space !== 'undefined') {
	if (opts.space === 'true') {
		opts.space = true;
	} else if (opts.space === 'false') {
		opts.space = false;
	} else if (/^\d+$/.test(opts.space)) {
		opts.space = parseInt(opts.space, 10);
	} else {
	  if (opts.space) {
		input.push(opts.space);
	  }
	  opts.space = true;
        }
}
```